Projects/Server/ExpansionInfo.cs

```diff
@@ -141,105 +145,33 @@ public class ExpansionInfo
-                    HousingFlags.None
-                ),
-                new ExpansionInfo(
-                    3,
-                    "Third Dawn",
-                    ClientFlags.Ilshenar,
-                    FeatureFlags.ExpansionUOTD,
-                    CharacterListFlags.ExpansionUOTD,
-                    HousingFlags.None
-                ),
-                new ExpansionInfo(
-                    4,
-                    "Blackthorn's Revenge",
-                    ClientFlags.Ilshenar,
-                    FeatureFlags.ExpansionLBR,
-                    CharacterListFlags.ExpansionLBR,
-                    HousingFlags.None
-                ),
-                new ExpansionInfo(
-                    5,
-                    "Age of Shadows",
-                    ClientFlags.Malas,
-                    FeatureFlags.ExpansionAOS,
-                    CharacterListFlags.ExpansionAOS,
-                    HousingFlags.HousingAOS
-                ),
-                new ExpansionInfo(
-                    6,
-                    "Samurai Empire",
-                    ClientFlags.Tokuno,
-                    FeatureFlags.ExpansionSE,
-                    CharacterListFlags.ExpansionSE,
-                    HousingFlags.HousingSE
-                ),
-                new ExpansionInfo(
-                    7,
-                    "Mondain's Legacy",
-                    new ClientVersion("5.0.0a"),
-                    FeatureFlags.ExpansionML,
-                    CharacterListFlags.ExpansionML,
-                    HousingFlags.HousingML
-                ),
-                new ExpansionInfo(
-                    8,
-                    "Stygian Abyss",
-                    ClientFlags.TerMur,
-                    FeatureFlags.ExpansionSA,
-                    CharacterListFlags.ExpansionSA,
-                    HousingFlags.HousingSA
-                ),
-                new ExpansionInfo(
-                    9,
-                    "High Seas",
-                    new ClientVersion("7.0.9.0"),
-                    FeatureFlags.ExpansionHS,
-                    CharacterListFlags.ExpansionHS,
-                    HousingFlags.HousingHS
-                ),
-                new ExpansionInfo(
-                    10,
-                    "Time of Legends",
-                    new ClientVersion("7.0.45.65"),
-                    FeatureFlags.ExpansionTOL,
-                    CharacterListFlags.ExpansionTOL,
-                    HousingFlags.HousingTOL
-                ),
-                new ExpansionInfo(
-                    11,
-                    "Endless Journey",
-                    new ClientVersion("7.0.61.0"),
-                    FeatureFlags.ExpansionEJ,
-                    CharacterListFlags.ExpansionEJ,
-                    HousingFlags.HousingEJ
-                )
-            };
+                var expansion = expansions[i];
+                if (expansion.ClientVersion != null)
```

Comment:
Almost missed this. We should always have braces around all if statements, even single line.